newinterface.ts

```diff
@@ -0,0 +1,137 @@
+// Primary entry points into this API: `browser()` and `feature()` factory
+// functions.
+
+export declare function browser(key: browserKey): Browser;
+// You'd use `browser()` to query for browsers, such as `browser("chrome")` or
+// `browser("safari")`.
+
+export declare function feature(id: string): Feature;
+// And `feature()` to query for specific features, as in
+// `feature("dotted.path.to.feature")`.
+
+// Before getting into the weeds, I should note that, ultimately, some methods
+// of the objects returned from the factory functions are going to provide
+// simplified feature support interpretations. They'll be subject to some
+// options, but thanks to the work eliminating `true` and `null` values from
+// BCD, we can be more restrictve in the API than we can be in the schema.
```

Comment:
One true and null are entirely gone from BCD, we'll remove it from the schema as well, right?

Reply:
Yes, that's correct. I haven't checked the stats lately—it's possible that this is already something we could do?